packages/actions/src/formatTag/action.yml

```
@@ -11,4 +11,4 @@ outputs:
     description: 'The semver string that was extracted from this tag'
 runs:
   using: node16
-  main: ../../dist/formatTag/index.js
+  main: ../../dist/formatTag/index.cjs
```

```suggestion
  main: ../../dist/formatTag/index.mjs
```

Can't we use `.mjs` instead here?

---

yeah, we can. Btw do we need cjs build for `/actions`? iirc when using tsup, we were using mjs only.

---

Yeah we don't in this case.